src/custom/components/Header/index.tsx

```diff
@@ -153,6 +153,7 @@ export default function Header() {
         <HeaderLinks>
           <StyledNavLink to="/swap">Swap</StyledNavLink>
           <StyledNavLink to="/about">About</StyledNavLink>
+          <StyledNavLink to="/rewards">Rewards</StyledNavLink>
```

Comment:
As I mentioned on slack, the page should now be named `Profile`.

```suggestion
          <StyledNavLink to="/profile">Profile</StyledNavLink>
```